lightning/src/ln/chan_utils.rs

```diff
@@ -587,24 +600,24 @@ impl LocalCommitmentTransaction {
 			},
 			their_sig: dummy_sig,
 			our_sig_first: false,
-			local_keys: TxCreationKeys {
+			holder_keys: TxCreationKeys {
 					per_commitment_point: dummy_key.clone(),
-					revocation_key: dummy_key.clone(),
-					a_htlc_key: dummy_key.clone(),
-					b_htlc_key: dummy_key.clone(),
-					a_delayed_payment_key: dummy_key.clone(),
+					broadcaster_revocation_key: dummy_key.clone(),
+					broadcaster_htlc_key: dummy_key.clone(),
+					countersignatory_htlc_key: dummy_key.clone(),
+					broadcaster_delayed_payment_key: dummy_key.clone(),
 				},
 			feerate_per_kw: 0,
 			per_htlc: Vec::new()
 		}
 	}
 
-	/// Generate a new LocalCommitmentTransaction based on a raw commitment transaction,
-	/// remote signature and both parties keys.
+	/// Generate a new HolderCommitmentTransaction based on a raw commitment transaction,
+	/// counterparty signature and both parties keys.
 	///
 	/// The unsigned transaction outputs must be consistent with htlc_data.  This function
 	/// only checks that the shape and amounts are consistent, but does not check the scriptPubkey.
-	pub fn new_missing_local_sig(unsigned_tx: Transaction, their_sig: Signature, our_funding_key: &PublicKey, their_funding_key: &PublicKey, local_keys: TxCreationKeys, feerate_per_kw: u32, htlc_data: Vec<(HTLCOutputInCommitment, Option<Signature>)>) -> LocalCommitmentTransaction {
+	pub fn new_missing_holder_sig(unsigned_tx: Transaction, their_sig: Signature, our_funding_key: &PublicKey, their_funding_key: &PublicKey, holder_keys: TxCreationKeys, feerate_per_kw: u32, htlc_data: Vec<(HTLCOutputInCommitment, Option<Signature>)>) -> HolderCommitmentTransaction {
```

Comment:
Several variables in this line need their/our swapped for the new names.

Reply:
Yes updated a bunch